.github/workflows/pytest.yml

```
@@ -0,0 +1,68 @@
+name: Pytest
+on: [push, pull_request, workflow_dispatch]
+
+env:
+  HUGEPAGES: 512
+
+jobs:
+  pytest:
+    strategy:
+      matrix:
+        test: ["cli", "state", "multi_gateway"]
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+        with:
+          # git submodule update --init --recursive
+          submodules: recursive
+
+      - name: Build images
+        run: |
+          SVC="ceph nvmeof-devel" make build
```

BTW, maybe it's a personal taste, but I find easier this syntax (command-options vs. options-command; this might be just [a cultural bias](https://en.wikipedia.org/wiki/Subject%E2%80%93verb%E2%80%93object_word_order) 😅 ):
```suggestion
          make build SVC="ceph nvmeof-devel"
```